front_end/ui/visual_logging/KnownContextValues.ts

```diff
@@ -3495,6 +3495,7 @@ export const knownContextValues = new Set([
   'time',
   'timeline',
   'timeline-alternative-navigation',
+  'timeline-frames',
```

Comment:
nit: Interesting, we haven't generally updated these and can probably omit this change.